### PR TITLE
add dubbo doc

### LINKED ISSUE #84: api_config.yaml中使用通用形式(dubbo request universality)无法调用dubbo
**What happened**:

参照`sample/dubbogo/proxy/api_config.yaml`里的配置和`docs/sample/dubbo-universality.md`里的方式，并没有返回期望的POST结果

**What you expected to happen**:
- Request

```bash
curl host:port/api/v1/test-dubbo/UserProvider/com.ic.user.UserProvider?method=GetUserByName&group=test&version=1.0.0 -X POST -d '{"types":["java.lang.String"],"values":"tc"}' --header "Content-Type: application/json"
```

- Response

```json
{
    "age": 18,
    "code": 1,
    "iD": "0001",
    "name": "tc"
}
```
    
**How to reproduce it (as minimally and precisely as possible)**:
1. 使用2020/12/22最新的develop代码`32fda76a` 编译 `sample/dubbogo/server/app`和`cmd/proxy/`
2. 使用`sample/dubbogo/server/config/`的配置启动server
3. 使用`sample/dubbogo/proxy/`的配置启动proxy
4. 所有配置中仅仅修改了zk的真实地址
5. 使用Get请求发送到`127.0.0.1:8888/api/v1/test-dubbo/user?name=tc`返回期望结果
6. 使用POST请求发送common请求，返回404

**Anything else we need to know?**:
1. 上述server和proxy运行在golang的pod中
2. 使用的镜像为docker hub上的`golang:1.14-stretch`, 即: go version go1.14.6 linux/amd64
3. server与proxy均使用该pod完成编译，再分别运行二进制文件
4. 运行结果如下:

```bash
# cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
VERSION_CODENAME=stretch
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

# uname -a
Linux my-hostname-8c9f6b5b6-t8jwx 3.10.0-957.el7.x86_64 #1 SMP Thu Oct 4 20:48:51 UTC 2018 x86_64 GNU/Linux

# ps -ef
UID         PID   PPID  C STIME TTY          TIME CMD
root          1      0  0 Dec15 ?        00:00:00 tail -f /etc/hosts
root       7098      0  0 09:44 ?        00:00:00 bash
root       7918   7098  0 09:49 ?        00:00:00 bash server-start.sh
root       7919   7918  0 09:49 ?        00:00:07 /go/bin/dubbo-proxy/dubbo-go-server-proxy
root       7936   7098  0 09:49 ?        00:00:00 bash proxy.sh
root       7937   7936  0 09:49 ?        00:00:06 /go/bin/dubbo-proxy/dubbo-go-proxy start -c /go/src/dubbo-proxy/sample/dubbogo/proxy/conf.yaml -a /go/src/dubbo-proxy/sample/dubbogo/proxy/api_config.yaml

# curl -i  http://127.0.0.1:8888/api/v1/test-dubbo/user?name=tc
HTTP/1.1 200 OK
Content-Type: application/json;charset=UTF-8
Date: Tue, 22 Dec 2020 01:58:27 GMT
Content-Length: 43

{"age":18,"code":1,"iD":"0001","name":"tc"}

# curl -i localhost:8888/api/v1/test-dubbo/UserProvider/com.ic.user.UserProvider?method=GetUserByName&group=test&version=1.0.0 -X POST -d '{"types":["java.lang.String"],"values":"tc"}' --header "Content-Type: application/json"
[1] 7987
[2] 7988
bash: -X: command not found
[2]+  Done                    group=test
root@golang-ecp-ewell-8c9f6b5b6-t8jwx:/go# HTTP/1.1 404 Not Found
Date: Tue, 22 Dec 2020 01:59:30 GMT
Content-Length: 18
Content-Type: text/plain; charset=utf-8

404 page not found
[1]+  Done                    curl -i localhost:8888/api/v1/test-dubbo/UserProvider/com.ic.user.UserProvider?method=GetUserByName
```

### PR DESCRIPTION
**Pull request by cityiron**
**What this PR does**:
Add doc for dubbo test, https://github.com/dubbogo/dubbo-go-proxy/issues/84

**Which issue(s) this PR fixes**:

Fixes #84

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/sample/dubbo.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-
-* [dubbo-query](dubbogo-query.md)
-* [dubbo-body](dubbogo-body.md)
+* [dubbo-query](dubbo-query.md)
+* [dubbo-body](dubbo-body.md)
+* [dubbo-uri](dubbo-uri.md)
+* [dubbo-mix](dubbo-mix.md)
 * [dubbo-universality](dubbo-universality.md)
 * [dubbo-error](dubbo-error.md)
 * [dubbo-incomplete](dubbo-incomplete.md)
```
Comment (zhangshen023) [inline docs/sample/dubbo.md:9]: image
add more comments for every doc, like "dubbo-query: ....."

File: docs/sample/mock.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Mock request
 
-## simple
+## Simple Demo
 
-### config
+### Config
```
Comment (zhangshen023) [inline docs/sample/mock.md:5]: api-config or conf? more clear?

**Review by zhangshen023 — APPROVED** [review]
hope you agree withmy a little tips

File: docs/sample/dubbo.md (hunk 1 above)
Comment (cityiron) [reply docs/sample/dubbo.md:9]: only name add ... ?

**Review by williamfeng323 — APPROVED** [review]

**Review by xiaoliu10 — APPROVED** [review]

**Pull request merged**